buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java

```diff
@@ -2,26 +2,33 @@
 
 import java.io.*;
 import java.util.LinkedList;
+import java.util.List;
 
+// todo: this class needs refactoring
 public class BuildCPIsHelper {
 
     public ProjectContext pc;
     public ProjectUtils utils ;
-    public BuildCPIsHelper(ProjectContext _pc) {
+
+    public NetworkConfig config;
+    public BuildCPIsHelper(ProjectContext _pc, NetworkConfig _config) {
         pc = _pc;
         utils = new ProjectUtils(pc);
+        config = _config;
     }
 
     public void createGroupPolicy() throws IOException {
 
         File groupPolicyFile = new File(String.format("%s/GroupPolicy.json", pc.devEnvWorkspace));
-        File devnetFile = new File(String.format("%s/config/dev-net.json", pc.project.getRootDir()));
+        File devnetFile = new File( pc.project.getRootDir() + "/" + config.getConfigFilePath());
```

Comment:
```suggestion
        File devnetFile = new File(pc.project.getRootDir() + "/" + config.getConfigFilePath());
```

BTW - isn't there a more standard way of concatenating sub-paths of a filesystem ?

Reply:
This is a quick fix to get things working, standardising the way we do paths will be part of the required refactor. The whole file needs refactoring, but the priority at the moment is to get it working.